Container.py

```diff
@@ -182,80 +181,3 @@ def construct_progressive_beam_patch(game_version: str, progressive_beams: bool)
         b('early_return_beam'),
     ]
     return instructions
-
-
-def construct_location_tracking_patch(game_version: str, player_state_offsets: List[int]) -> List[int]:
-    from ppc_asm.assembler.ppc import cmpwi, lwz, r5, r6, r7, r8, r9, r10, r11, r12, nop, ble, b, addi, li, rlwinm, or_, stw
-    # r5 = current amount of unknown item 1
-    # r6 = player_state (loaded from previous)
-    # r7 - r10 = potential target offsets
-    # r11 = target bit position
-    # r12 = selected target register with correct group
-
-    # Load current amount of unknown item 1 into r5
-    # Load each potential target into r7 through r10
-    # Determine target offset and set that into r12
-    # Determine bit position and set that into r11
-
-    UNKNOWN_ITEM_1_ID = misc_item_table["UnknownItem1"].id
-    UNKNOWN_ITEM_2_ID = misc_item_table["UnknownItem2"].id
-    HEALTH_REFILL_ID = misc_item_table["HealthRefill"].id
-
-    def get_current_amount_offset(item_id: int):
-        return calculate_item_offset(item_id) + 0x0
-
-    def get_current_capacity_offset(item_id: int):
-        return calculate_item_offset(item_id) + 0x4
-
-    instructions = [
-        *_load_player_state_to_r6(game_version),

```

Comment:
I think this will actually need to go in the progressive beam patch if I recall correctly, I think it assumes that this register was already set for player state address

Reply:
might be a good idea to check previous version that hadn't this patch and compare with it